tests/data/pdo-stmt-execute.php

```diff
@@ -19,10 +19,25 @@ public function execute(PDO $pdo)
         $stmt->execute(['adaid' => 1]); // prefixed ":" is optional
         assertType('PDOStatement<array{email: string, 0: string, adaid: int<0, 4294967295>, 1: int<0, 4294967295>}>', $stmt);
 
+        $stmt = $pdo->prepare('SELECT email, adaid FROM ada WHERE email = :email');
+        assertType('PDOStatement', $stmt);
+        $stmt->execute([':email' => 'email@example.org']);
+        assertType('PDOStatement<array{email: string, 0: string, adaid: int<0, 4294967295>, 1: int<0, 4294967295>}>', $stmt);
+
+        $stmt = $pdo->prepare('SELECT email, adaid FROM ada WHERE email <=> :email');
```

Comment:
mysql syntax für NULL values in prepared statements, see https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_equal-to